### PR TITLE
Add .editorconfig file to enforce editor rules

### PR DESCRIPTION
**Pull request by bolatovumar**
This PR adds `.editorconfig` file which enforces some rules which if not followed break the build. Specifically it enforces Unix-style newlines, new line ending in every file, usage of spaces for indentation with the tab size being equal to two spaces and the trimming of trailing whitespace.

Remaining commits are there to get the build to be green again.

### CODE REVIEW

**Comment by 0xmountaintop** [comment]
I am not sure whether an .editorconfig is desired.
not all the editors support .editorconfig natively, and it may require knowledge on such a tool.

I believe the travis lint check is enough?

**Comment by bolatovumar** [comment]
@HAOYUatHZ I don't see why it wouldn't be desired. If your editor supports it then it will just use the config so you don't have to manually make changes like adding newlines at the end of every file for Travis build to pass. If your editor doesn't support it then you still lose nothing. There is nothing extra required from the user.

**Comment by aantonop** [comment]
@bolatovumar Happy to merge this, but because you mixed edits and a new file in your PR, it now has conflicts. I suggest you separate PRs. In each PR do only one thing, in only one file. It's easier for the editors to merge smaller and more focused PRs that don't cause conflicts.

Can you split this into two PRs, or resolve the conflicts please?

**Comment by bolatovumar** [comment]
@aantonop the reason I mixed in other stuff into this PR was because the build was failing prior to my PR and I really don't like submitting PRs which break the build so I had to fix up the build issues. I will rebase and resolve the conflicts.

**Comment by bolatovumar** [comment]
@aantonop ping

File: 02_getting_started.asciidoc [hunk 1]
```diff
@@ -35,9 +35,9 @@ Blockchains, especially open blockchains like Bitcoin, attempt to minimize or el
...
 
-Every user must consider their own technical skills before deciding what type of lightning wallet to use. Those with strong technical skills should use a lightning wallet that puts all of the components under the direct control of the user. Those with less technical skill but a desire to control their funds, should choose a _non-custodial_ lightning wallet, even if some of the components (other than the keystore) rely on some trusted third parties. 
-Often the trust in those cases relates to privacy. 
-If users decide to outsource some functionality to a third party they usually give up some privacy as the third party will learn some information.
+Every user must consider their own technical skills before deciding what type of lightning wallet to use. Those with strong technical skills should use a lightning wallet that puts all of the components under the direct control of the user. Those with less technical skill but a desire to control their funds, should choose a _non-custodial_ lightning wallet, even if some of the components (other than the keystore) rely on some trusted third parties.
+Ofter the trust in those cases relates to privacy.
```
Comment (yahiheb) [inline 02_getting_started.asciidoc]:
```suggestion
Often the trust in those cases relates to privacy.
```

File: 02_getting_started.asciidoc [hunk 2]
```diff
@@ -35,9 +35,9 @@ Blockchains, especially open blockchains like Bitcoin, attempt to minimize or el
...
+If users decides to outsource some functionality to a third party they usually give up some privacy as the third party will learn some information.
```
Comment (yahiheb) [inline 02_getting_started.asciidoc]:
```suggestion
If users decide to outsource some functionality to a third party they usually give up some privacy as the third party will learn some information.
```

**Review by yahiheb — CHANGES_REQUESTED** [review]

**Comment by bolatovumar** [comment]
@yahiheb the changes you are suggesting are not related to this PR. This PR is specifically about adding an `.editorconfig` file to make it easier enforce certain rules and fixing up any issues that are currently breaking the build. You should open another PR which fixes the issues you pointed out.

**Comment by yahiheb** [comment]
The typos were made in this PR https://github.com/lnbook/lnbook/pull/141/commits/9f54a72d05f3d633c8b64f95af0e3f06668923f9

**Comment by bolatovumar** [comment]
@yahiheb ok, thanks, fixed now.

**Comment by aantonop** [comment]
Thanks @bolatovumar !

**Pull request merged**